### PR TITLE
Change votes to show next period. Add tooltip with further info

### PR DESCRIPTION
**Pull request by timjrobinson**
# Description

- Add new vue for Vote Information in the Gauge Voting table. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

- Open Gauge Voting table at `/#/vebal`
- Check that tooltip looks correct and has this week + next weeks votes + change in votes, and is the correct color. 

## Visual context

https://www.loom.com/share/19a19f4ecea34d4995326db09e15c883

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue [hunk 1]
```diff
@@ -90,16 +92,11 @@ const columns = ref<ColumnDefinition<VotingGaugeWithVotes>[]>([
...
   {
     name: t('veBAL.liquidityMining.table.nextPeriodVotes'),
-    accessor(gauge) {
-      const normalizedVotes = scale(new BigNumber(gauge.votes), -18);
-      return fNum2(normalizedVotes.toString(), {
-        style: 'percent',
-        maximumFractionDigits: 2
-      });
-    },
+    acessor: 'id',
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue:95]:
```suggestion
    accessor: 'id',
```

File: src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue [hunk 2]
```diff
@@ -90,16 +92,11 @@ const columns = ref<ColumnDefinition<VotingGaugeWithVotes>[]>([
...
     align: 'right',
-    id: 'poolGaugeVotes',
-    sortKey: gauge => Number(gauge.votes),
+    id: 'nextPeriodVotes',
+    Cell: 'nextPeriodVotesCell',
+    sortKey: gauge => Number(gauge.votesNextWeek),
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue:99]: `gauge.votesNextWeek` seems to throw a TS error, are you sure this property is available.

File: src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue [hunk 3]
```diff
@@ -240,6 +237,11 @@ function redirectToPool(gauge: VotingGaugeWithVotes) {
           />
         </div>
       </template>
+      <template #nextPeriodVotesCell="gauge">
+        <div v-if="!isLoading" class="px-6 py-4">
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue:241]: I'm not sure why the `align` property isn't doing this but please fix the alignment:
```suggestion
        <div v-if="!isLoading" class="px-6 py-4 text-right">
```

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue (hunk 3 above)
Comment (timjrobinson) [reply src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue]: This makes the text in the popup alight along the right hand side. Why did you want this changed?

Comment (timjrobinson) [reply src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue]: It looks like this:
tooltip

**Review by garethfuller — APPROVED** [review]

**Pull request merged**